src/screens/NoteListScreen.js

```diff
@@ -287,7 +287,7 @@ const mapStateToProps = (state) => {
  */
 const mapDispatchToProps = (dispatch) => {
     return {
-        deleteNote: (noteId) => dispatch(actions.notes.deleteNote({ noteId }))
```

Comment:
@adrianhall
To my understanding objects are different before and after app sync setup due to which nodeId is not found after 
app sync setup. it's been sometime I have worked on it. Don't remember the exact issue in detail.

if you want to reproduce it, one way could be doing the setup as per doc and once it's done try deleting an item from the list.